stemcell_builder/stages/bosh_rust/apply.sh

```
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -e
+
+base_dir=$(readlink -nf "$(dirname "${0}")/../..")
+source "${base_dir}/lib/prelude_apply.bash"
+source "${base_dir}/lib/prelude_bosh.bash"
+
+# rustup verifies downloaded toolchain integrity via its own signing chain; TLS on sh.rustup.rs is the trust anchor for the installer itself
+curl_five_times "${chroot}/tmp/rustup-init.sh" "https://sh.rustup.rs"
```

Shellcheck (0.11.0)

[warning] chroot is referenced but not assigned.

(SC2154)